src/api/java/appeng/api/recipes/ICraftHandler.java

```diff
@@ -30,7 +30,7 @@
 import appeng.api.exceptions.RecipeException;
 import appeng.api.exceptions.RegistrationException;
 
-
+@Deprecated
 public interface ICraftHandler
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the missing @deprecated Javadoc tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AMissingDeprecatedCheck)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1133)